src/Microsoft.DotNet.Interactive/Parsing/PolyglotSyntaxParser.cs

```diff
@@ -188,12 +188,16 @@ void AppendNode(SyntaxNode node)
                     case DirectiveParameterNode parameterNode when subcommandNode is null:
                         directiveNode.Add(parameterNode);
                         break;
+
                     case DirectiveParameterValueNode valueNode when subcommandNode is null:
                         directiveNode.Add(valueNode);
                         break;
+                    case DirectiveParameterValueNode valueNode when subcommandNode is not null:
+                        subcommandNode.Add(valueNode);
+                        break;
 
-                    case DirectiveNameNode nameNode when subcommandNode is not null:
-                        subcommandNode.Add(nameNode);
+                    case DirectiveParameterNode parameterNode when subcommandNode is null:
```

Comment:
Is this a duplicate of the `case` on line 188 above?

Reply:
It is! Nice catch.